src/net/eth.ts

```diff
@@ -135,6 +140,37 @@ export default class Eth extends Api {
             Eth.loadContracts(dispatch, web3);
             // fetch addresses
             await dispatch(fetchEthAddress());
+            // Ethereum event for inbound channel contracts
+            dispatch(subscribeEthereumEvents());
+            
+            //Obtain transaction list
+            let stateData:any = store.getState();
+            if(stateData && stateData.transactions && stateData.transactions.transactions.length>0) {
+                let pendingTxCount = pendingEventTransactions(stateData.transactions.transactions);
+                if (pendingTxCount == 0) {
```

Comment:
Use `===`.

Reply:
Thanks, I've adjusted the code to reflect your suggestions.